### PR TITLE
[bitnami/solr] Chart standardized

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juanariza@vmware.com>

**Description of the change**

This PR standardizes the `bitnami/solr` chart to be in line with the rest of the catalog.

**Benefits**

Standardization.

**Possible drawbacks**

Changes the values structure forcing a major bump, check the README.md for more information.

**Applicable issues**

N/A

**Additional information**

Blocked by https://github.com/bitnami/charts/pull/8623 (major bump in dep)

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/solr/templates/statefulset.yaml [hunk 1]
```diff
@@ -81,95 +96,66 @@ spec:
...
+        {{- if .Values.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
         {{- end }}
       containers:
         - name: solr
           image: {{ include "solr.image" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
-          {{- if .Values.lifecycleHooks }}
-          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.lifecycleHooks "context" $) | nindent 12 }}
-          {{- end }}
```
Comment (migruiz4) [inline bitnami/solr/templates/statefulset.yaml:148]: Why removing `lifecycleHooks`?

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Please check my comments below.

In addition, please add the following missing value:
- `kubeVersion`

File: bitnami/solr/templates/NOTES.txt [hunk 2]
```diff
@@ -15,76 +16,60 @@ APP VERSION: {{ .Chart.AppVersion }}
...
+To connect to your Solr cluster from outside the cluster, perform the following steps:
 
 {{- if contains "NodePort" .Values.service.type }}
 
+1. Obtain the NodePort IP and ports:
+
     export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
     export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "common.names.fullname" . }})
-    echo "Go to ${NODE_IP}:${NODE_PORT}"
+    echo "Solr URL: ${NODE_IP}[${NODE_PORT}]"
```
[the remark this answers is not in the document]
Comment (juan131) [reply bitnami/solr/templates/NOTES.txt:43]: The rest of examples were using brackets too. But I don't have a strong opinion on this

File: bitnami/solr/templates/statefulset.yaml (hunk 1 above)
Comment (juan131) [reply bitnami/solr/templates/statefulset.yaml:148]: My mistake

**Review by migruiz4 — APPROVED** [review]
LGTM!

**Pull request merged**